docs/src/pages/guides/publish-to-npm.md

```diff
@@ -1,9 +1,10 @@
 ---
 layout: ~/layouts/MainLayout.astro
 title: Publish to NPM
+modifiedDate: '2021-09-06 @ 11:28:21'
```

Comment:
curious, what's the reason for adding this?

Reply:
Whoops.. So sorry for the delay, this was auto-added by an extension of mine and I missed it 🤐